int32 count
string status
---
bool finish